tests/e2e/e2e_suite_test.go

```diff
@@ -15,11 +15,18 @@ var ctx context.Context
 var token string
 var clusterID string
 
+func GetToken() string {
+	return token
+}
+
+func GetClusterID() string {
```

Comment:
Could you advise what is better - to use getter here? Or just call the function once (as in line 22/28) and just use it inside the tests?
@nirarg @bardielle @sagidayan

Reply:
What's the purpose for the function?

Reply:
just getting the clusterID
same is dont for the token.
I think it is more nit that way, but I don't mind changing

Reply:
changed back to be without the getter